velox/exec/tests/HashJoinTest.cpp

```diff
@@ -2803,6 +2803,60 @@ TEST_F(HashJoinTest, semiProject) {
       .run();
 }
 
+TEST_F(HashJoinTest, semiProjectWithNullKeys) {
```

Comment:
I think you could move this to MultiThreadedHashJoinTest

Reply:
I think so. Other semiProjectXxx tests are in HashJoinTest, hence, I added this one here. Maybe we can move them all to MultiThreadedHashJoinTest in a follow-up PR.